Code/src/main/java/nl/utwente/group10/ui/components/blocks/input/DisplayBlock.java

```diff
@@ -54,26 +55,28 @@ protected DisplayBlock(CustomUIPane pane, String fxml) {
 
         this.loadFXML(fxml);
 
-        inputAnchor = new InputAnchor(this, pane);
-        anchorSpace.getChildren().add(inputAnchor);
+        inputAnchor = new InputAnchor(this, HindleyMilner.makeVariable());
+        inputAnchor.layoutXProperty().bind(inputSpace.widthProperty().divide(2));
+        inputSpace.getChildren().add(inputAnchor);        
+        
+        //Make sure inputSpace is drawn on top.
+        BorderPane borderPane = (BorderPane) inputSpace.getParent();
```

Comment:
I disagree, this BorderPane is only relevant because it is the parent, thus I find it more logical to obtain it by requesting the parent.